.docker/gitpod/Dockerfile

```diff
@@ -1,12 +1,16 @@
 # This is the Dockerfile for `leanprovercommunity/mathlib:gitpod`.
+# As well as elan and mathlib, we also install leanblueprint and all of its dependencies.
+
+# This container does not come with a pre-installed version of mathlib.
+# When a gitpod container is spawned, elan will be updated and mathlib will be downloaded:
+# see the .gitpod.yml file for more information.
 
-# gitpod doesn't support multiple FROM statements, (or rather, you can't copy from one to another)
-# so we just install everything in one go
 FROM ubuntu:jammy
 
 USER root
 
-RUN apt-get update && apt-get install sudo git curl git bash-completion python3 python3-requests -y && apt-get clean
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt-get install sudo git curl git bash-completion texlive texlive-xetex latexmk graphviz graphviz-dev python3 python3-pip python3-requests -y && apt-get clean
```

Comment:
Why do we want `texlive` for mathlib? This makes the image quite a bit bigger.

Reply:
Maybe we should make two tags: one that includes a copy of leanblueprint and its dependencies (such as TeXLive), and one that does not.

Reply:
This is because we want it for blueprints. But maybe that should be an extra tag/dockerhub repo